src/calendars/poland.js

```diff
@@ -666,26 +666,26 @@ let dates = year => {
     },
     {
       "key": "blessedMaryOfJesusTheGoodShepherdVirgin",
-      "type": Types[6],
+      "type": Types.OPT_MEMORIAL,
       "moment": moment.utc({ year: year, month: 10, day: 25 }),
       "data": {}
     },
     {
       "key": "blessedRafalChylinskiPriest",
-      "type": Types[6],
+      "type": Types.OPT_MEMORIAL,
       "moment": moment.utc({ year: year, month: 11, day: 2 }),
       "data": {}
     },
     {
       "key": "saintBarbaraVirginAndMartyr",
-      "type": Types[6],
+      "type": Types.OPT_MEMORIAL,
       "moment": moment.utc({ year: year, month: 11, day: 4 }),
       "data": {}
     },
     // [pejulian] Needs confirmation: In Poland this day is a feast
```

Comment:
@jarosz, could you confirm this please?

Reply:
@emagnier, this one might also be a separate issue then—what do you think?

Reply:
yes same things

Reply:
Moved to issue #116.